library/std/src/os/freebsd/fs.rs

```diff
@@ -76,9 +76,9 @@ impl MetadataExt for Metadata {
     fn as_raw_stat(&self) -> &raw::stat {
         // The methods below use libc::stat, so they work fine when libc is built with FreeBSD 12 ABI.
         // This method would just return nonsense.
-        #[cfg(freebsd12)]
+        #[cfg(any(freebsd12, freebsd13))]
         panic!("as_raw_stat not supported with FreeBSD 12 ABI");
-        #[cfg(not(freebsd12))]
+        #[cfg(not(any(freebsd12, freebsd13)))]
```

Comment:
If build.rs defines both `freebsd12` and `freebsd13` then this shouldn't be needed.